Add dashboard blurg (#78)
Contenda-Team/Contenda-landing-page: src/posts/dashboards.md

---
layout: "../layouts/BlogPost.astro"
title: "Pain free content management with Contenda dashboards"
slug: dashboard-content-management
description: "A home for all your content"
author: "Grady Salzman"
added: "Jul 6 2023"

> tags: ["dashboards", "release"]

Reviewer: ```suggestion
tags: ["dashboards", "announcements"]
```

---
layout: "../layouts/BlogPost.astro"
title: "Pain free content management with Contenda dashboards"
slug: dashboard-content-management
description: "All content deserves a loving ~~dashboard~~, I mean _home_. "
author: "Grady Salzman"
added: "Jul 6 2023"
tags: ["dashboards", "announcements"]
---

Before dashboards, Contenda users had to manually keep track of their `blog_id`. We searched our inbox for the email confirmation and clicked the provided link. This made it a pain to use Contenda over time because you always had to manually search for it in your inbox. 

With your Contenda dashboard you can, 

 - See all your posts

<video controls autoplay loop>
  <source src="/blogimages/ClickBlogPost.mp4" type="video/mp4">
</video>

 - Access user settings to generate api keys or manage your workspace

<video controls autoplay loop>
  <source src="/blogimages/Settings.mp4" type="video/mp4">

> </video>

Reviewer 2: The dashboard is now live at [app.contenda.co](url), give it a try for free!.

> </video>

Reviewer 2: Add another paragraph about workspace: We've noticed that some of you prefer to have a shared space where you can see and interact with each other's posts. In response to this, we've created a workspace feature that allows you to invite others simply by sharing a code. It's a convenient way to collaborate and stay connected within a group!

---
layout: "../layouts/BlogPost.astro"
title: "Pain free content management with Contenda dashboards"
slug: dashboard-content-management

> description: "All content deserves a loving ~~dashboard~~, I mean _home_. "

Reviewer 3: ```suggestion
description: "All content deserves a loving dashboard, I mean home. "
```
<img width="749" alt="Screen Shot 2023-07-06 at 12 48 42 PM" src="https://github.com/Contenda-Team/Contenda-landing-page/assets/8921721/8295e09f-9eee-487e-8c4d-d37bb4f43e7d">
lol description does not format with markdown for whatever reason

author: "Grady Salzman"
added: "Jul 6 2023"
tags: ["dashboards", "announcements"]
---

Before dashboards, Contenda users had to manually keep track of their `blog_id`. We searched our inbox for the email confirmation and clicked the provided link. This made it a pain to use Contenda over time because you always had to manually search for it in your inbox. 

> Now with dashboards, users have a home for all their posts and can easily retrieve them. This dashboard will also show posts made by other users in your workspace. Allowing you to easily collaborate with others on your posts without needed to share links or user credentials. Managing user and workspace settings also lives on the dashboard making it easily accessible. 

Reviewer 3: ```suggestion
Now with dashboards, users have a home for all their posts. The dashboard also provides a home for posts made by other users in your workspace! You can now easily collaborate and coordinate with others on your content creation workflow without needing to pass along your user credentials (since [Share Pear](https://contenda.co/blog/public-sharing/) is read only). The dashboard also houses user and workspace settings. How cozy! 
```
might be too home-y feel free to drop some of it
